.github/workflows/e2e.yml

```diff
@@ -129,7 +129,7 @@ jobs:
   e2e-tests-notify:
     needs: [ e2e, e2e_management, e2e_s3_gcs, e2e_s3 ]
     runs-on: ubuntu-latest
-    if: github.ref == 'refs/heads/master'
+    if: (success() || failure()) && github.ref == 'refs/heads/master'
```

Comment:
Isin't it the same as `always()`?

Reply:
`always()` additionally ignores manual cancellation of workflow.